File: solutions/0026-remove-duplicates-from-sorted-array/solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if(nums==null || nums.length==0){
            return 0;
        }
        int i =0,j=1;
        int prev = nums[0];
        while(i<nums.length){
            if(prev!=nums[i]){
                prev = nums[i];
                nums[j] = nums[i];

                j++;
            }
                i++;
        }
      return j;
    }
}

File: solutions/0026-remove-duplicates-from-sorted-array/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int prev = nums[0];
        int index=1;
        for(int& num:nums){
            if(num!=prev){
                prev = num;
                nums[index]=num;
                index++;
            }
        }
        return index;

    }
};
